addon/components/input-validator.ts

```diff
@@ -0,0 +1,71 @@
+import Component from '@ember/component';
+// @ts-ignore: Ignore import of compiled template
+import layout from '../templates/components/input-validator';
+import { computed } from '@ember-decorators/object';
+import { bool, reads } from '@ember-decorators/object/computed';
+import { observes } from '@ember-decorators/object';
```

Comment:
more of just a general thought than an actionable change on this PR... but once 3.10 lands with built-in decorator support (presumably), are we going to have change these imports to use ember core's decorators rather than getting them from the `@ember-decorators` addon, do you know?

Reply:
I don't believe we have to, but we should